### ISSUE TITLE
Problems opening with window HEIGHT in window mode

### ISSUE THREAD

**Issue by paramecie** [rgriebl/brickstore#12]
1) It doesn't memorize the window HEIGHT (so in window mode, not full screen).
It always open it with a huge height here.

2) Then on the "splash screen" (open recent files...), I cannot resize its height.
I can resize the width, not the height.

3) But when a file has been opened, then I can resize the height.

Later on, if I then close the application and reopen it, it'll open with this far bigger height, not the last one...
But width is fine, saved fine, and can be resized any time.

**Comment by paramecie** [comment]
Just to help the diagnostic, I used regedit and removed the Geometry entry here:

HKEY_USERS\......\Software\BrickStore\BrickStore\MainWindow\Layout

![image](https://user-images.githubusercontent.com/32078594/103595214-2d60ab80-4efb-11eb-88c0-d5248e629d55.png)

I opened BrickStore and it opened as the first time, with a very weird size.

The fact is I have 2 horizontal monitors here, both 1920 x 1080 (24")

When BS opened the first time, its width was approx 1.5 monitor, maybe around 3000px large, (so, not 1 screen or 2, somewhere in between), and the height, I just can't mesure it, probably the double of my screen height, or maybe 1.5x ?

In short, maybe it's disturbed by the 2 screens config ?

![image](https://user-images.githubusercontent.com/32078594/103595568-0fe01180-4efc-11eb-8f37-8cc2a9a0ed2c.png)

**Comment by paramecie** [comment]
Confirm! 2021.1.2 apparently fixed this, and saves both window state and size properly! Thank you.
